target/linux/rockchip/patches-5.10/201-net-phy-Add-driver-for-Motorcomm-YT85xx-PHYs.patch

```diff
@@ -0,0 +1,472 @@
+From d113e3b5649f4919276be0ab904e1360d6b84aab Mon Sep 17 00:00:00 2001
+From: hmz007 <hmz007@gmail.com>
+Date: Mon, 5 Jul 2021 17:03:00 +0800
+Subject: [PATCH 2/4] net: phy: Add driver for Motorcomm YT85xx PHYs
```

Comment:
There is an upstream driver I'd much rather use

https://github.com/torvalds/linux/blob/master/drivers/net/phy/motorcomm.c